cmd/headscale/cli/preauthkeys.go

```diff
@@ -96,6 +108,15 @@ var listPreAuthKeys = &cobra.Command{
 				reusable = fmt.Sprintf("%v", key.GetReusable())
 			}
 
+			var aclTags string
+
+			if len(key.AclTags) > 0 {
+				for _, tag := range key.AclTags {
+					aclTags += "," + tag
+				}
+				aclTags = strings.TrimLeft(aclTags, ",")
+			}
```

Comment:
This `len` check is strictly not necessary, this should work:
```suggestion
			aclTags := ""

			for _, tag := range key.AclTags {
				aclTags += "," + tag
			}
			aclTags = strings.TrimLeft(aclTags, ",")
```

Reply:
Since the loop will not run if the list has 0 items, its safe to not check it.

Reply:
:+1: ty this was me not understanding go at this point of the journey haha